src/test/objectstore/BitAllocator_test.cc

```diff
@@ -420,28 +420,28 @@ TEST(BitAllocator, test_bmap_alloc)
       alloc->free_blocks(i, 1);
     }
 
-		int64_t blk_size = 1024;
-		std::vector<AllocExtent> extents = std::vector<AllocExtent>
-					(alloc->size(), AllocExtent(-1, -1));
+    int64_t blk_size = 1024;
+    std::vector<AllocExtent> extents = std::vector<AllocExtent>
+          (alloc->size(), AllocExtent(-1, -1));
 
-		ExtentList *block_list = new ExtentList(&extents, blk_size);
+    ExtentList *block_list = new ExtentList(&extents, blk_size);
 
-		allocated = alloc->alloc_blocks_dis(alloc->size()/2, block_list);
-		bmap_test_assert(allocated == alloc->size() / 2);
+    allocated = alloc->alloc_blocks_dis(alloc->size()/2, block_list);
+    bmap_test_assert(allocated == alloc->size() / 2);
```

Comment:
nit, might want to use `ASSERT_EQ(alloc->size() / 2, allocated)` instead. easier to understand the test result by inspecting the failure report than `ASSERT_EQ(true, (x))`.